2021/09/deploying-dotnet-5-app-iis.md

```diff
@@ -0,0 +1,74 @@
+---
+author: "Juan Pablo Ventoso"
+date: 2021-09-16
+title: "Deploying a .NET 5 app on IIS"
+tags:
+- dotnet
+- csharp
+- iis
+---
+
+![Puzzle](/blog/2021/09/deploying-dotnet-5-app-iis/puzzle-cropped.jpg)
+
+[Photo](https://flic.kr/p/4riBj8) by [Christian Cable](https://www.flickr.com/photos/nexus_icon/)
+
+.NET 5 is been around for a few years now, after being released at the [.NET Conf 2020](/blog/2020/11/dotnet-5-released-net-conf-2020/), containing the best of both worlds: .NET Core, including multi-platform support and several performance improvements, and .NET Framework, including Windows desktop development support with WPF and Windows Forms (UWP is also supported, but not officially yet).
+
+A .NET Core-based project can be published into any platform (as long as we’re not depending on libraries targeted to .NET Framework), allowing us to save costs by hosting in Linux servers and increasing performance for scalability. But most developers are still using Windows with Internet Informacion Services (IIS) as the publishing target: This might be related to the almost 20 years of history of .NET Framework, comparing to the relatively short history of .NET Core, launched in 2016.
```

Comment:
It sounds like you're saying Linux is a more performant platform than Windows for hosting .NET apps. But I don't think that's true, is it?

Maybe you mean that because Linux has no license cost, it's cheaper to deploy multiple servers in parallel, and thus get better performance for the same cost?

Or that Windows costs increase with the # of CPU cores and processors, whereas on Linux you can scale up core count for free?

Reply:
Yeah, that's what I meant, now that I re-read it, it was not clear. I rephrased it, does it sound better now? I can change it if needed.